### ISSUE TITLE
Don't encourage shebangs for MS-DOS .bat files

### ISSUE THREAD

**Issue by mcandre** [duggan/shlint#14]
MS-DOS .bat files do not allow shebangs; please skip such warnings for `.bat` files.

```
$ cat test.bat
echo howdy
$ checkbashisms test.bat
script test.bat does not appear to have a #! interpreter line;
you may get strange results
```

We should probably skip this warning for PowerShell (`.ps1`) files as well.
